mllib/src/main/scala/org/apache/spark/ml/util/Instrumentation.scala

```diff
@@ -85,6 +85,13 @@ private[ml] class Instrumentation[E <: Estimator[_]] private (
   }
 
   /**
+   * Logs the value with customized name field.
+   */
+  def logNamedValue(name: String, num: Long): Unit = {
```

Comment:
Just a note for posterity, I thought about whether this should be a Double to support arbitrary values (maybe in future we want to log some training metrics such as log loss or other error, for example), but then the formatting would be off for integers. So if we need that in future we can just create an overloaded method.